requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # For running doctests (during "python setup.py test")
 Sphinx >= 1.4.5
+sphinx_bootstrap_theme
```

Comment:
Hum.... well, I thought that spinx_rtd_theme was a requirement for the bootstrap_theme. Maybe pip already solves it without our explicit "request".
So, I think that's all ok. Tks ;)